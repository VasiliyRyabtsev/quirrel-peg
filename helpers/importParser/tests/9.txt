#directive1 #directive2:yes
/* werwert */  #complex-directive:no
local a = require("x.nut")
return a.x
