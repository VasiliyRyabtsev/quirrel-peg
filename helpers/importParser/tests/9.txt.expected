#directive1
#directive2:yes
#complex-directive:no
